graphene_django_extras/fields.py

```diff
@@ -172,7 +172,7 @@ def __init__(
             *args,
             **kwargs,
     ):
-        self.resolve_func = kwargs['resolve_queryset']
+        self.resolve_func = kwargs.get('resolve_queryset')
```

Comment:
We need this resolve function always. Why this condition?

Reply:
Removing this....it was temp change for one of the model where I didn't inherit Graphql